[ICDS] Added logging information when 0 in Program Summary report

lwyszomi opened the pull request:
Hi @Rohit25negi @calellowitz 

I added logs when all indicators equal 0 for Program Summary report. 

TICKET: https://dimagi-dev.atlassian.net/browse/ICDS-547

Need QA: No
Environment: ICDS
Locally Tested: yes

Commit:
[ICDS] Added logging information when 0 in Program Summary report

calellowitz reviewed (changes requested):
I spoke to Anirudh about this, and he suggested combining this with removal of the feature flag. Additionally, he requested the logging go to sentry instead of a file. If you are unsure how to get it in sentry, I think you can use `notify_exception` to do so, although there may be a more direct way.

Commit:
move logging to sentry

Commit:
clean code

lwyszomi commented:
Hi @calellowitz @Rohit25negi 

I updated PR, I didn't use ```notify_exception``` but I used the same logger ``notify``` also I removed code which is using when a user does not have access to feature flag.

Please let me know if you have any questions.

Regards,
Łukasz
